pkg/util/timeutil/zoneinfo.go

```diff
@@ -30,7 +30,14 @@ func LoadLocation(name string) (*time.Location, error) {
 	switch loweredName {
 	case "local", "default":
 		loweredName = "utc"
-		name = "UTC"
```

Comment:
according to the function comment, it seems like we still want to be setting `name = "UTC"` so that the call to `time.LoadLocation` below does what we want. but if that's the case, i'm surprised CI would pass